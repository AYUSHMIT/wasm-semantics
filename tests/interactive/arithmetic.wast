i32.const 5
i32.const 7
i32.add

i32.const 5
i32.const 7
i32.sub

i32.const 15
i32.const 3
i32.mul

i32.const 3
i32.const 15
i32.div_u

i32.const 2
i32.const 15
i32.div_u

i32.const 3
i32.const 15
i32.rem_u

i32.const 2
i32.const 15
i32.rem_u

i32.const 20
i32.const 18
i32.and

i32.const 20
i32.const 18
i32.or

i32.const 20
i32.const 18
i32.xor

i32.const 0
i32.eqz

i32.const 3
i32.eqz

i32.const 3
i32.const 3
i32.eq

i32.const 3
i32.const 4
i32.eq

i32.const 3
i32.const 3
i32.ne

i32.const 3
i32.const 4
i32.ne
