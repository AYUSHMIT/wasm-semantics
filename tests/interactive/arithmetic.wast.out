<generatedTop>
  <k>
    .
  </k>
  <stack>
    < i32 > 1 : < i32 > 0 : < i32 > 0 : < i32 > 1 : < i32 > 0 : < i32 > 1 : < i32 > 6 : < i32 > 22 : < i32 > 16 : < i32 > 1 : < i32 > 0 : < i32 > 7 : < i32 > 5 : < i32 > 45 : < i32 > 2 : < i32 > 12 : .Stack
  </stack>
</generatedTop>
